apps/user/interfaces/user.interface.ts

```diff
@@ -1,75 +1,107 @@
-export interface UserI {
-    id?: string,
-    username?: string,
-    email?: string,
-    firstName?: string,
-    lastName?: string,
-    isEmailVerified?: boolean,
-    clientId?: string,
-    clientSecret?: string,
-    supabaseUserId?: string,
-    userOrgRoles?: object
+export  interface UInvitation {
+  orgRoles?: OrgRole[];
+  status?: string;
+  id?: string;
+  orgId?: string;
+  organisation?: Organisation;
+  userId?: string;
 }
-
-export interface InvitationsI {
-    id: string,
-    userId: string,
-    orgId?: string,
-    organisation?: object
-    orgRoleId?: string,
-    status: string,
-    email?: string
-    orgRoles: string[]
+export interface Organisation {
+  id: string;
+  name: string;
+  logoUrl: string;
 }
-
-export interface UserEmailVerificationDto{
- email:string
- username?:string
-}
-
-export interface userInfo{
-    email: string,
-    password: string,
-    firstName: string,
-    lastName: string,
-    isPasskey: boolean
-}
-
-export interface AddPasskeyDetails{
-    password: string,
-}
-
-export interface UserWhereUniqueInput{
-    id?: string
+export interface OrgRole {
+  id: string;
+  name: string;
+  description: string;
 }
 
-export interface UserWhereInput{
-    email?: string
-}
-export interface UpdateUserProfile {
-    id: string,
+export interface UserI {
```

Comment:
Please use meaningful names for interface.

Reply:
Resolved.